keps/sig-node/3545-improved-multi-numa-alignment/README.md

```diff
@@ -141,7 +144,7 @@ When `prefer-closest-numa-nodes` policy is enabled, we need to retrieve informat
 Right now Topology manager discovers Node layout using [CAdvisor API](https://github.com/google/cadvisor/blob/master/info/v1/machine.go#L40).
 
 We will need to extend the `MachineInfo` struct with a `Distances` field which will describe the distance between a given NUMA node and other NUMA nodes in the system.
-This is already implemented in `cadvisor` by this [patch](https://github.com/google/cadvisor/pull/3179) but it is not yet present in any of the released versions. 
+This is already implemented in `cadvisor` by this [patch](https://github.com/google/cadvisor/pull/3179) but it is not yet present in any of the released versions.
```

Comment:
is it still the case? it seems cadvisor >= 0.46 includes the change, and in kube main branch we are vendoring now v0.50.0. Perhaps we can mention this.

Reply:
rephrased the sentence.